### PR TITLE
[FT]Add user can allocate check different stakeholders

### PR DESCRIPTION
**Pull request by thoth-ky**
#### Problem
A user who has team access to another user's healthchecks account is notified on the status of all the jobs defined in that account
#### Solution
Fixed the process of adding a user to team access so that user assigns a specific check to which the user has access.
#### Screenshots
This is all the checks in the profile
Owner
This shows only checks assigned to this user. The rest of the checks in this profile are hidden from this user.
Assigned User

[Pivotal Tracker Card](https://www.pivotaltracker.com/story/show/158532883)

### CODE REVIEW

File: hc/accounts/tests/test_profile.py [hunk 1]
```diff
@@ -11,13 +11,14 @@ def test_it_sends_set_password_link(self):
...
+        assert response.status_code == 302
 
         # profile.token should be set now
         self.alice.profile.refresh_from_db()
         token = self.alice.profile.token
-        ### Assert that the token is set
+        # Assert that the token is set
+        self.assertTrue(token)
 
         ### Assert that the email was sent and check email content
```
Comment (miritih) [inline hc/accounts/tests/test_profile.py]: Use one # for block comments

File: hc/accounts/tests/test_profile.py [hunk 2]
```diff
@@ -32,16 +33,18 @@ def test_it_sends_report(self):
...
         member_emails = set()
         for member in self.alice.profile.member_set.all():
             member_emails.add(member.user.email)
 
-        ### Assert the existence of the member emails
-
+        # Assert the existence of the member emails
         self.assertTrue("frank@example.org" in member_emails)
 
         ###Assert that the email was sent and check email content
```
Comment (miritih) [inline hc/accounts/tests/test_profile.py]: Each line of a block comment starts with a # and a single space. Consider changing this one too

**Review by miritih — CHANGES_REQUESTED** [review]

**Review by loicemeyo — APPROVED** [review]
It's a great work done...and for going a mile and working on the tests as well.

File: hc/accounts/tests/test_profile.py (hunk 2 above)
Comment (simiyu1) [inline hc/accounts/tests/test_profile.py]: Well commented code... However, you could also add the test that the email has been sent and checking the email content.

**Review by miritih — APPROVED** [review]

File: hc/accounts/tests/test_profile.py (hunk 2 above)
Comment (thoth-ky) [reply hc/accounts/tests/test_profile.py]: Done

Comment (thoth-ky) [reply hc/accounts/tests/test_profile.py]: Assertions to check if invites have been sent added

**Review by simiyu1 — APPROVED** [review]
Clean code!

**Review by karanjaE — APPROVED** [review]

**Pull request merged**